client/.sass-lint.yml

```diff
@@ -0,0 +1,56 @@
+####################
```

Comment:
@peeb Have you discussed sass linting with @gabemart?

* How do I run this? I don't see any deps added to package.json. Do I need a global (npm) install or is it part of sass itself?
* Where are these rules from or did you write them specifically for lumen?
* If we want this in the CI, should we still use `output-file`?
* Is `max-warnings` what we want? Or should we do the same as with eslint, so a lint error results in a broken build?
* If there are lots of errors/warnings for existing sass files, should this be part of a separate PR, with the errors/warnings fixed?